packages/@blockprotocol/type-system-web/package.json

```diff
@@ -18,6 +18,10 @@
   "scripts": {
     "lint:tsc": "tsc --noEmit"
   },
+  "devDependencies": {
+    "@local/tsconfig": "0.0.0-private",
+    "typescript": "4.8.2"
```

Comment:
Both `@local/tsconfig` and `typescript` need to be present in all TS workspaces.